package.json

```diff
@@ -16,6 +16,7 @@
     "Alex Lande"
   ],
   "scripts": {
+    "dist": "webpack --config ./webpack.config.js",
```

Comment:
GLOBAL: Never prefix with `./`. Windows is compatible as long as you don't do that. (Internal `/` are fine).